boards/arm/nrf51_ble400/doc/nrf51_ble400.rst

```diff
@@ -0,0 +1,248 @@
+.. _nrf51_ble400:
+
+Waveshare BLE400
+#################
+
+Overview
+********
+
+BLE400 is a mother board designed for BLE4.0 Bluetooth 2.4G Wireless Module. Used together with Core board.
+
+Supported Features
+==================
+
+Mother board
```

Comment:
Motherboard, one word

Reply:
But I think "Board" would be better

Reply:
Look - I am not native english speaker :) so just used terms from https://www.waveshare.com/wiki/BLE400 where motherboard is separate words. I had doubts about it so finally used original manufacturer terminology.

Reply:
I see, just saw their wiki. It's not a big deal, I will not block the merge because of this,

Reply:
Also I think Moverboard term should be better than just "board" to avoid confusion and explicitly distinguish it from Core board used together with it.

Reply:
Agreed now that I saw the Wiki

Reply:
I too thought the term "mother board" seemed wrong, but that's what their site calls it (though they're not native English speakers).